pkgs/os-specific/linux/nvidia-x11/generic.nix

```diff
@@ -118,6 +121,27 @@ let
     nativeBuildInputs = [ perl nukeReferences which libarchive ]
       ++ optionals (!libsOnly) kernel.moduleBuildDependencies;
 
+    env.libnvidia_nscq = let
+      inherit (stdenv.hostPlatform) isAarch64 isLinux isx86_64;
+      args =
+        if isx86_64
+        then {
+          arch = "x86_64";
+          hash = nscqSha256_64bit;
+        }
+        else if isAarch64
+        then {
+          arch = "aarch64";
+          hash = nscqSha256_aarch64;
+        }
+        else throw "libnvidia_nscq is not supported on ${stdenv.hostPlatform.system}";
+      src = fetchzip {
+        inherit (args) hash;
+        url = "https://developer.download.nvidia.com/compute/nvidia-driver/redist/libnvidia_nscq/linux-${args.arch}/libnvidia_nscq-linux-${args.arch}-${nscqVersion}-archive.tar.xz";
+      };
+    in
+      lib.optionalString isLinux "${src}";
```

Comment:
I believe this will break the `i686-linux` `legacy_390` driver, since it will throw an error for that architecture